### DISCUSSION TITLE
Multiple "Set-Cookie" fields in getServerSideProps?

### DISCUSSION THREAD

**Discussion by v-tran-cna** [vercel/next.js#22363] [Help]
In getServerSideProps I need to set multiple cookies on client device. I coded: 
`ctx.res.setHeader("set-cookie", "access-token=1");`
`ctx.res.setHeader("set-cookie", "refresh-token=1");`

And result: 
![image](https://user-images.githubusercontent.com/62643677/108579101-f84bd380-7357-11eb-84d5-f1635893b800.png)

I found in https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js, that's because 
![image](https://user-images.githubusercontent.com/62643677/108579127-19142900-7358-11eb-8574-eef30dc670c8.png)

How can I  set multiple cookies in getServerSideProps, maybe like this?:
![image](https://user-images.githubusercontent.com/62643677/108579341-04846080-7359-11eb-8b1e-16a0e760b3bb.png)

**Comment by maticzav** [comment]
Try using [`nookies`](http://github.com/maticzav/nookies).

**Comment by sannjayy** [comment]
##### Do like This:

```
res.setHeader('Set-Cookie', [
    cookie.serialize(
        'access', user.tokens.access, {
            httpOnly: true,
            secure: process.env.NODE_ENV !== 'development',
            maxAge: 60 * 30,
            sameSite: 'strict',
            path: '/api/'
        }
    ),
    cookie.serialize(
        'refresh', user.tokens.refresh, {
            httpOnly: true,
            secure: process.env.NODE_ENV !== 'development',
            maxAge: 60 * 60 * 24,
            sameSite: 'strict',
            path: '/api/'
        }
    )
]);
```
